src/m_menu.c

```diff
@@ -5830,7 +5830,7 @@ boolean M_Responder (event_t* ev)
 		// Don't bind movement and turning to mouse wheel. It needs to
 		// be impossible to input a one-frame of movement automatically
 		// in speedrunning.
-		if ((ch == MOUSE_BUTTON_WHEELUP || ch == MOUSE_BUTTON_WHEELDOWN) &&
```

Comment:
Maybe it's time for `M_IsMouseWheel(int mouseb)` in `m_input`? :^)

Reply:
> Maybe it's time for `M_IsMouseWheel(int mouseb)` in `m_input`? :^)

Good idea, but can probing wait till after this PR.